src/main.h

```diff
@@ -260,10 +260,10 @@ bool IsSuperBlock(CBlockIndex* pIndex);
 
 double GetEstimatedNetworkWeight(unsigned int nPoSInterval = 40);
 double GetAverageDifficulty(unsigned int nPoSInterval = 40);
-//double GetEstimatedTimetoStake(unsigned int nPoSInterval = 40, double dConfidence = 0.8);
 // Note that dDiff cannot be = 0 normally. This is set as default because you can't specify the output of
 // GetAverageDifficulty(nPosInterval) = to dDiff here.
-double GetEstimatedTimetoStake(double dDiff = 0.0, double dConfidence = 0.8);
+// The defeult confidence is 1-1/e which is the mean for the geometric distribution.
+double GetEstimatedTimetoStake(double dDiff = 0.0, double dConfidence = 0.632120558829);
```

Comment:
Can we define the new default value for `dConfidence` in a constant to formalize it?

Reply:
Sure.